### PR TITLE
Some minor fixes at patient issues

### PR DESCRIPTION
**Pull request by matrix-amiel**
Hi.

Here have 2 fixes in patient issues.
1. Disabling a option of writing duplicate coding at interface/patient_file/summary/add_edit_issue.php.
Prevent this situation - 
image

2. Refresh issues screen after adding/updating encounter connection to get right result.
(This is the single solution that I've found.. I open to another suggestions.. )

Thanks 
 Amiel

### CODE REVIEW

File: interface/patient_file/summary/add_edit_issue.php [hunk 1]
```diff
@@ -564,8 +564,11 @@ function set_related(codetype, code, selector, codedesc) {
  var s = f.form_diagnosis.value;
  var title = f.form_title.value;
  if (code) {
-  if (s.length > 0) s += ';';
-  s += codetype + ':' + code;
+     //disabled duplicate codes
+     if (s.indexOf(codetype + ':' + code) == -1){
+         if (s.length > 0) s += ';';
+         s += codetype + ':' + code;
+     }
```
Comment (bradymiller) [inline interface/patient_file/summary/add_edit_issue.php]: This fix looks good.

File: interface/patient_file/problem_encounter.php [hunk 2]
```diff
@@ -380,6 +381,10 @@ function doclick(pfx, id) {
  }
  if ($alertmsg) echo "alert('" . addslashes($alertmsg) . "');\n";
 ?>
+// reload the parent screen (after saving openemr is not available)
+$('#problem_encounter').submit(function (e) {
+    opener.reloadAfterSecond();
+});
```
Comment (bradymiller) [inline interface/patient_file/problem_encounter.php]: Wouldn't something like this work(using same mechanism as add_edit_issue.php callback)?
https://github.com/matrix-israel/openemr/blob/cae3ed0675d11226804f3fd9286b1f3aa6390897/interface/patient_file/summary/add_edit_issue.php#L352-L357

Comment (matrix-amiel) [inline interface/patient_file/problem_encounter.php]: Thank you!
It's good solution.
I fixed it.

Amiel

**Comment by bradymiller** [comment]
Just brought this into the codebase. Thanks for the fixes! -brady

**Pull request merged**